BallisticTables.json

```diff
@@ -0,0 +1,26 @@
+{"82mm_a3": [
+    {"charge": 0, "table": [
```

Comment:
I don't think javascript exception handling is particularly strong, but you can store it to a variable and then null check the variable. Though just stating min/max ranges would be easier since it would require no lookup and no null checking at all.